_posts/2019-12-26-policy-gradients.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title:  "Understanding Policy Gradients"
+date:   2019-12-26 22:19:00 +0530
+categories: reinforcement-learning, policy-gradients
+---
+
+# Understanding Policy Gradients
+In reinforcement learning, we have an agent interacting with the environment. The agent acts, the environment gives feedback in the for of rewards. The agent wants to maximize the total rewards it gets.
+
+The goal goal of policy gradients: make an agent (a Neural Network, to be concrete) that maximizes its expected return. To be precise, the agent must: for every state it might be in, assign a probability distribution for actions it might take, so that it gets the maximum possible return on average from then onwards.
+
```

Comment:
goal is written twice here, change that.
I think you can elaborate on the introduction a bit more.
Add a little more intro on RL as well as maybe contrast Policy gradients with Q-value based algorithms.